infra/base-images/base-builder/install_go_CodeIntelligenceTesting.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -eux
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Install CodeIntelligenceTesting Go.
+## Require at least Go1.4 to boostrap.
+cd /tmp
+git clone --depth=1 -b dev.libfuzzer.18 https://github.com/CodeIntelligenceTesting/go.git .go-CodeIntelligenceTesting
+cd .go-CodeIntelligenceTesting/src
+# Disable tests, at least one of which erroneously fails.
+sed -i '/^exec .* tool dist test -rebuild "$@"/ s/./#&/' run.bash
+./all.bash
+
+# Replace original Go with the one from CodeIntelligenceTesting.
+rm -rf /root/.go
+mv -f /tmp/.go-CodeIntelligenceTesting /root/.go
+
+# Install go114-fuzz-build with the new Go.
+rm -rf "$GOPATH/"
+mkdir -p "$GOPATH/"
+go install github.com/mdempsky/go114-fuzz-build@latest
+ln -s "$GOPATH/bin/go114-fuzz-build" "$GOPATH/bin/go-fuzz"
+
```

Comment:
nit: delete this line.

Reply:
Removed in [this commit](https://github.com/google/oss-fuzz/pull/7240/commits/19104a5462a4215b1ba6df269d59b6e31e2e5b7d).